mkt/api/urls.py

```diff
@@ -41,7 +41,8 @@ def include_version(version):
     return include('mkt.api.v%d.urls' % version, **kwargs)
 
 
-urlpatterns = patterns('',
+urlpatterns = patterns(
+    '',
```

Comment:
We could, its not great that the first argument is always ''. Does this block the PR for you? Trying to avoid changing too much code in the PR as it is.